### PR TITLE
Update CHANGELOG for v3 release

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
Update CHANGELOG.md for v3.0.0 release.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
N/A

**Checklist:**

1. Have you successfully run `npm run build:release` locally? Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by LeviSklaroff — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0] - 2022-03-30
+Amazon Chime SDK JS 3.0.0 is here !! 🎉🎉🎉
```
Comment (simmkyu) [inline CHANGELOG.md:9]: `Chime SDK JS 3.0.0` => `Chime SDK for JavaScript v3`

(We used `v2` in the [v2 migration guide](https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_2_0.html). An alternative option is just `3`, `3.0.0`, `version 3`.)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+Amazon Chime SDK JS 3.0.0 includes the following major changes:
+
+- Each user’s audio and video devices are now fully decoupled from their connection to the WebRTC media session. 
+  A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly and without interrupting their WebRTC connection.
+- The 3.0 client libraries now use the standardized WebRTC metrics for all browsers.
+- Add support for aws-sdk js v3 for messaging session to help avoid using multiple versions in a single application
```
Comment (simmkyu) [inline CHANGELOG.md:17]: `aws-sdk js v3` => `for the AWS SDK for JavaScript v3`

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0] - 2022-03-30
+Amazon Chime SDK JS 3.0.0 is here !! 🎉🎉🎉
+
+Amazon Chime SDK JS 3.0.0 includes the following major changes:
+
+- Each user’s audio and video devices are now fully decoupled from their connection to the WebRTC media session. 
+  A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly and without interrupting their WebRTC connection.
```
Comment (simmkyu) [inline CHANGELOG.md:15]: Remove `and`?

`the user can switch devices instantly and without` =>
`the user can instantly switch devices without interrupting`

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- The 3.0 client libraries now use the standardized WebRTC metrics for all browsers.
+- Add support for aws-sdk js v3 for messaging session to help avoid using multiple versions in a single application 
+  and minimize application footprint.
+- Deprecated support for Safari 12, and Plan B in Session Description Protocol (SDP) negotiations  
+
+Please refer to the [migration guide](https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_3_0.html) for
```
Comment (simmkyu) [inline CHANGELOG.md:21]: How about explaining the below list? e.g.,

> Below is a list of all changes in the Chime SDK for JavaScript 3.0.0. Please read the [Migration from SDK v2 to SDK v3](https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_3_0.html) for more information.

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0] - 2022-03-30
+Amazon Chime SDK JS 3.0.0 is here !! 🎉🎉🎉
+
+Amazon Chime SDK JS 3.0.0 includes the following major changes:
```
Comment (simmkyu) [inline CHANGELOG.md:11]: nit: You can put a one-line summary.

> Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the messaging session.

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+- Each user’s audio and video devices are now fully decoupled from their connection to the WebRTC media session. 
+  A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly and without interrupting their WebRTC connection.
+- The 3.0 client libraries now use the standardized WebRTC metrics for all browsers.
```
Comment (simmkyu) [inline CHANGELOG.md:16]: What are `client libraries`? How about?

> The SDK v3 now publishes the standardized WebRTC metrics for all supported browsers.

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Add support for aws-sdk js v3 for messaging session to help avoid using multiple versions in a single application 
+  and minimize application footprint.
+- Deprecated support for Safari 12, and Plan B in Session Description Protocol (SDP) negotiations
```
Comment (simmkyu) [inline CHANGELOG.md:19]: 1st and 2nd = Complete sentence
3rd = Add (verb)
4th = Deprecated (past tense verb)

Make all points consistent.

**Comment by simmkyu** [comment]
Amazon Chime SDK for JavaScript v3 is here! 🎉🎉🎉

Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the messaging session.

- **Device management**: ...
- **WebRTC metrics**: The SDK now publishes the standardized WebRTC metrics for all supported browsers.
- **Messaging session**: The messaging session now supports the AWS SDK for JavaScript v3.
- **Dropping support**: The SDK deprecates support for Safari 12 and Plan B in Session Description Protocol (SDP) negotiations.

Below is a list of all changes in the Chime SDK for JavaScript 3.0.0. Please read the [Migration from SDK v2 to SDK v3](https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_3_0.html) for more information.

File: CHANGELOG.md [hunk 8]
```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-30
 
+Amazon Chime SDK JS v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the 
+messaging session.
+
+- **Device management:** Decouple each user’s audio and video devices from their connection to the WebRTC media
```
Comment (simmkyu) [inline CHANGELOG.md:15]: nit: How about removing terms like `WebRTC` and `connection`? From the user perspective, they simply join or start a meeting session.

e.g.,

> Decouple audio and video device management from the meeting sessions. For example, a user can select their preferred devices on a video preview page and continue using the same devices to join the session. After joining the session, a user can instantly switch devices without interrupting the ongoing meeting session.

File: CHANGELOG.md [hunk 9]
```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+- **Device management:** Decouple each user’s audio and video devices from their connection to the WebRTC media 
+  session. A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly without interrupting their WebRTC connection.
+- **WebRTC metrics:** Publish the standardized WebRTC metrics for all supported browsers.
+- **Messaging session:** Add support for AWS SDK for JavaScript v3 for messaging session to help avoid using multiple 
+  versions in a single application and minimize application footprint.
+- **Dropping support:** Deprecate support for Safari 12, and Plan B in Session Description Protocol (SDP) negotiations  
+
+Below is a list of all changes in the Chime SDK for JavaScript 3.0.0. Please refer to the [Migraton guide from v2 to v3](https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_3_0.html) for
```
Comment (simmkyu) [inline CHANGELOG.md:23]: oops sorry. `3.0.0` => `v3` to match one in the first paragraph.

File: CHANGELOG.md [hunk 10]
```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+Amazon Chime SDK JS v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the 
+messaging session.
+
+- **Device management:** Decouple each user’s audio and video devices from their connection to the WebRTC media 
+  session. A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly without interrupting their WebRTC connection.
+- **WebRTC metrics:** Publish the standardized WebRTC metrics for all supported browsers.
+- **Messaging session:** Add support for AWS SDK for JavaScript v3 for messaging session to help avoid using multiple
```
Comment (simmkyu) [inline CHANGELOG.md:19]: > Support AWS SDK for JavaScript v3 in the messaging session.

(`To help avoid using multiple versions ...` part does not make sense for AWS SDK v2 users?)

File: CHANGELOG.md [hunk 11]
```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-30
 
+Amazon Chime SDK JS v3 is here !! 🎉🎉🎉
```
Comment (simmkyu) [inline CHANGELOG.md:10]: `Amazon Chime SDK JS v3` => Amazon Chime SDK for JavaScript v3`

File: CHANGELOG.md [hunk 12]
```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the 
+messaging session.
+
+- **Device management:** Decouple each user’s audio and video devices from their connection to the WebRTC media 
+  session. A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly without interrupting their WebRTC connection.
+- **WebRTC metrics:** Publish the standardized WebRTC metrics for all supported browsers.
+- **Messaging session:** Add support for AWS SDK for JavaScript v3 for messaging session to help avoid using multiple 
+  versions in a single application and minimize application footprint.
+- **Dropping support:** Deprecate support for Safari 12, and Plan B in Session Description Protocol (SDP) negotiations
```
Comment (simmkyu) [inline CHANGELOG.md:21]: Missing `.` and move some words around

> Deprecate Safari 12 support and Plan B in Session Description Protocol (SDP) negotiations.

**Review by simmkyu — APPROVED** [review]

**Pull request merged**